AdyenCard/Components/Card/CardComponentDelegate.swift

```diff
@@ -21,13 +21,13 @@ public protocol CardComponentDelegate: AnyObject {
     func didChangeCardBrand(_ value: [CardBrand]?, component: CardComponent)
 
     /// Called when shopper submits a card in `CardComponent`.
-    /// - Parameter value: Last 4 digits of card.
+    /// - Parameter lastFour: Last 4 digits of card.
```

Comment:
I think the param name is still `value` ?

Reply:
lastFour is what is shown to the code user, but value is more internal to the function, right?